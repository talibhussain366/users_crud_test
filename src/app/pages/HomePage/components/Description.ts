import styled from 'styled-components/macro';

export const Description = styled.div`
  color: white;
  font-size: 12px;
`;
