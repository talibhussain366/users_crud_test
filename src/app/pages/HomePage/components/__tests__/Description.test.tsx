import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { Description } from '../Description';

const shallowRenderer = createRenderer();

describe('<Description />', () => {
    it('<Description />', () => {
        shallowRenderer.render(<Description />);
        const renderedOutput = shallowRenderer.getRenderOutput();
        expect(renderedOutput).toMatchSnapshot();
    });
});
