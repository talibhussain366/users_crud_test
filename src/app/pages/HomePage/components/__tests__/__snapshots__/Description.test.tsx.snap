// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> <Description /> 1`] = `
<div
  className="Description-sc-1ve6h2t-0 liDJcf"
/>
`;
